lib/App/cpanminus/script.pm

```diff
@@ -869,7 +869,7 @@ Options:
   -n,--notest               Do not run unit tests
   --test-only               Run tests only, do not install
   -S,--sudo                 sudo to run install commands
-  --installdeps             Only install dependencies
+  --installdeps             Only install 'test' dependencies
```

Comment:
No, that's an explanation why `test` is included in some situations and not in others.